packages/e2e-cypress/src/helpers/cct-dev-server/index.ts

```diff
@@ -11,10 +11,9 @@ async function exportQuasarConfig(bundler: AvailableBundlers): Promise<any> {
   let quasarAppPackage = `@quasar/app-${bundler}`;
 
   if (bundler === 'webpack') {
-    try {
-      await exec('npm ls @quasar/app-webpack');
```

Comment:
Can you remove `exec` import and promisifying, since we're not using it anymore?

Reply:
That's what I've done. Check the red (removed) lines above

Reply:
Done!